src/storage/kv.js

```diff
@@ -0,0 +1,184 @@
+const api = require('../api');
+const { getNamespaceByName } = require('./storage');
+
+/**
+ * Gets namespaces associated with the account
+ * @param {*} options
+ */
+async function getKeys({ accountId, nameSpace } = {}) {
+  const account = accountId || process.env.CLOUDFLARE_ACCOUNT_ID;
+
+  if (!account) {
+    throw 'You must provide an account ID';
+  }
+
+  const nameSpaceId = await getNamespaceByName({
+    accountId: account,
+    name: nameSpace
+  });
+
+  if (!nameSpaceId) {
+    throw 'Name space does not exist';
+  }
+
+  let result = await api.cfApiCall({
+    url: `/accounts/${account}/storage/kv/namespaces/${nameSpaceId}/keys`,
+    method: 'GET'
+  });
+
+  if (result.success && result.result) {
+    result = result.result;
+  }
+
+  return result;
+}
+
+/**
+ * Gets namespaces associated with the account
+ * @param {*} options
+ */
+async function getAllKeysAndValues({ accountId, nameSpace } = {}) {
+  const account = accountId || process.env.CLOUDFLARE_ACCOUNT_ID;
+
+  if (!account) {
+    throw 'You must provide an account ID';
+  }
+
+  const keys = await getKeys({
+    accountId: account,
+    nameSpace: nameSpace
```

Comment:
Is this by name?

Reply:
I just didn't want it to be mistaken with key name or anything else, so I made it nameSpace.